dashboard/app/controllers/followers_controller.rb

```diff
@@ -115,19 +73,10 @@ def redirect_url
   end
 
   def load_section
-    p request.path
-    if params[:section_code].blank?
-      if request.path != student_user_new_path(section_code: params[:section_code])
```

Comment:
i removed this block (lines 118-126) because it was never being executed. we only wanted to execute this path if a user makes a POST request to `/join/:section_code` with no section code, but it is no longer executed because this line (120) will never be true. more explicitly, if `params[:section_code` is empty, we are in this situation:

`request.path` == `/join` (for `POST /join/:section_code`)
`student_user_new_path(section_code: params[:section_code])` == `/join`

so `request.path != student_user_new_path(section_code: params[:section_code])` will always be false.

Reply:
statement above is still true, but i needed to add back a portion of the previous logic:

```
return if params[:section_code].blank?
```